WaterLog_Backend/Models/TankLevelsEntry.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WaterLog_Backend.Models
+{
+    public class TankLevelsEntry
+    {
+        [Key]
+        public int Tank_Id { get; set; }
```

Comment:
Remove the underscore, keep the property names consistent and PascalCase

Reply:
done